acs-category-theory-notes.tex

```diff
@@ -1,28 +1,61 @@
-\documentclass[a4paper, 12pt]{article}
+%%%%%%%%%%%%%%%%%%%%%%%% NOTES TO CONTRIBUTORS %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% - try to stick to a minimal package set unless you need them. In other words,
+% don't use packages you don't understand simply because you're used to using
+% them.
+% - $$ eqn $$ is less universal than \[ eqn \] or \begin{equation} eqn \end{eqn}
+% - Don't use the math environment just to center things. That's what the center
+% environment is for.
+% - If you have other stylistic guidelines you want other poeple to keep in
+% mind, put them here
+
+
+% In the future we may want to break this into many subfiles (see the subfiles
+% package) so that we're not all working on a single monolithic document that
+% takes a long time to compile.
+
+\documentclass[a4paper, 12pt, oneside]{book}
+
 \usepackage{a4wide}
 \usepackage{nth}
-\usepackage{fourier} % Better font
-\usepackage{inconsolata} % mono spaced font
+% The "better font" is not scalable, also less standard
```

Comment:
I'm pretty sure Fourier is a scalable font. It's based on Utopia by Adobe. Perhaps I was missing `
\usepackage[T1]{fontenc}`. It comes with the texlive distribution, so I don't see how it's not standard.

Reply:
To be fair, I can always recompile it myself with whatever font I prefer. So in that regard the default font should presumably be whatever the majority of people are happy with.
I would still suggest that a significant proportion of people do not have high dpi displays, and therefore argue in favour of font that renders well on computer screens as the default.